File: magic-dialog/src/main/java/com/jaceed/android/magicdialog/OnDialogFragmentInteraction.java
package com.jaceed.android.magicdialog;

import android.view.View;

/**
 * Created by Jacee.
 * Date: 2022.04.07
 */
public interface OnDialogFragmentInteraction {

    void onDialogNegativeClick(View v);

    void onDialogPositiveClick(View v);

    default void onDialogNeutralClick(View v) {
    }

}

File: MagicDialog/src/main/java/com/baicizhan/framework/common/magicdialog/OnDialogFragmentInteraction.kt
package com.baicizhan.framework.common.magicdialog

import android.view.View

interface OnDialogFragmentInteraction {
    fun onDialogNegativeClick(v: View)
    fun onDialogPositiveClick(v: View)
    fun onDialogNeutralClick(v: View) {}
}